pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
 
 import (
```

Comment:
Let's review binding to make sure we handle all the edge cases that PV/PVC binder code handles, for example:
* There are two pre-provisioned VolumeSnapshot objects in two different namespaces. pointing to the same VolumeSnapshotContent object
* There are two pre-provisioned VolumeSnapshotContent objects pointing to  the same VolumeSnapshot object.
* During binding, before you can update both objects, something changes.
* Do we need Phase on VolumeSnapshotContent?
* etc.